ansible/roles_ocp_workloads/ocp4_workload_quarkus_workshop_user/tasks/pre_workload.yml

```diff
@@ -0,0 +1,84 @@
+---
+# Implement your Pre Workload deployment tasks here
+
+- name: Ensuring username is defined
+  fail:
```

Comment:
You could use `assert` here to check all conditions at once.

Reply:
Done!